### DISCUSSION TITLE
Daisy 2.52: Please advise, hover, active, focus not working on menu items

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#2427] [Q&A] [answered]
Hi

I need some advise here.

Here is the reproduction. My hover, active, focus, active classes are not working on the second menu items.

https://play.tailwindcss.com/THgNs21hJK

Thanks in advance.

**Comment by saadeghi** [comment]
Here's your example with version `2.52` and it's working 🤔  
https://play.tailwindcss.com/TCnAf9gbWs?file=config  

Can you explain more about the issue?

**Reply by Mashiane** [reply]
![Background](https://github.com/saadeghi/daisyui/assets/5799306/615e29f7-d32a-41fd-8dcd-0c41ff0b7e2b)


I have set hover:bg-transparent on each of the li items, however this is not working. Or perhaps its necessary that I add an anchor and assign that? Is it necessary to have an anchor?

**Reply by Mashiane** [reply]
Sorry I think my title was not clear enough..

**Reply by saadeghi** [reply, marked as answer]
Use those hover, active, focus classes on the div inside `<li>` (not the `<li>` itself). Because that div is getting the background color on hover
